src/main/java/org/cbioportal/legacy/web/GeneAssistantController.java

```diff
@@ -0,0 +1,45 @@
+package org.cbioportal.legacy.web;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import org.cbioportal.application.assistant.GeneAssistantService;
+import org.cbioportal.legacy.model.GeneAssistantResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@Validated
+@RestController()
+@ConditionalOnProperty(name = "spring.ai.enabled", havingValue = "true")
```

Comment:
wondering if we need to set CORS restrictions somewhere for these endpoints? So only users of the website can use m